templates/sshd_config.erb

```diff
@@ -80,9 +80,9 @@ AuthorizedKeysCommandUser <%= @sshd_authorized_keys_command_user %>
 HostbasedAuthentication <%= @sshd_hostbasedauthentication %>
 # Change to yes if you don't trust ~/.ssh/known_hosts for
 # RhostsRSAAuthentication and HostbasedAuthentication
-#IgnoreUserKnownHosts no
```

Comment:
This line should stay. We decided that these lines should be kept so that you can see what the default value is, which is helpful in debugging ssh problems.